pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -333,8 +341,10 @@ message PodSandboxStatus {
     optional LinuxPodSandboxStatus linux = 6;
     // Labels are key value pairs that may be used to scope and select individual resources.
     map<string, string> labels = 7;
-    // Annotations is an unstructured key value map that may be set by external
-    // tools to store and retrieve arbitrary metadata.
+    // Unstructured key value map that may be set by external tools to store
+    // and retrieve arbitrary metadata.
+    // The value of this field MUST be identical to that of the corresponding
+    // PodSandboxConfig used to instantiate the pod sandbox this status represents.
     map<string, string> annotations = 8;
```

Comment:
I see, I was confused by the distinction between api (user-set) annotations and the system-set per-container annotations. IIUC the annotations on the pod are the api annotations, and the container level annotations are system-set, right? I think it might be worth distinguishing between these explicitly, either in a comment or the field name.

If that's the case, it seems like the annotations on the PodSandbox and PodSandboxStatus messages are redundant?